pkg/application/types.go

```diff
@@ -0,0 +1,30 @@
+package application
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Application
+type App struct {
```

Comment:
Ideally the API about the CRDs should not be part of odo and managed as a separate project. This is the reason why @claprun created this project - https://github.com/snowdrop/component-api that we will use to manage CRD API (till it could become a k8s project)